CONTRIBUTING.md

```diff
@@ -275,3 +275,24 @@ see <https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html>.
 
 With this, you should now have a working development setup! See
 [above](#building-and-testing-miri) for how to proceed working on Miri.
+
+### Syncing the miri subrree
+
+#### Updating from the rustc repo
+
+One time setup:
+
+Add your own miri fork as `miri-your-fork` to the rustc repo:
+
+```
+git remote add miri-your-fork git@github.com:your-name/miri.git
+```
+
+Every time:
+
+```
+ulimit -s 60000
+wget https://raw.githubusercontent.com/gitgitgadget/git/fe2e4819b869725f870cd3ce99f1f8150fe17dc1/contrib/subtree/git-subtree.sh
+sh git-subtree.sh push --prefix src/tools/miri/ miri-your-fork miri-subtree-sync
+rm git-subtree.sh
```

Comment:
I think we should just recommend to store that script somewhere. Removing it each time sounds silly?

Reply:
XD I thought you wanted to avoid that

Reply:
Yeah by storing it in the git repo.^^